packages/dds/tree/api-report/tree.api.md

```diff
@@ -1705,6 +1713,9 @@ export function stackTreeFieldCursor<TNode>(adapter: CursorAdapter<TNode>, root:
 // @internal
 export function stackTreeNodeCursor<TNode>(adapter: CursorAdapter<TNode>, root: TNode): CursorWithNode<TNode>;
 
+// @public
+export type StoredKey = string;
```

Comment:
No opposition to being more verbose, but I hesitate to specify "schema" here, since this really describes a key persisted in _data_, not just schema. If we wanted to be more verbose, I might go with something like `StoredFieldKey`.

Reply:
StoredFieldKey aligns well with our conventions (adding "field" to key as a clarifier, and adding "Stored" as a clarifier).